src/conventional/sonde_tac2ioda.py

```diff
@@ -724,10 +728,12 @@ def getDayHour(str):
     :param str: The string to parse
     :return: (day, hour)
     """
+    global scale_wspd
```

Comment:
confused why this specification of `global` is needed, as I read the code `scale_wspd` is already a global in this routine